language/en/extimgaslink.php

```diff
@@ -23,5 +23,11 @@
 }
 
 $lang = array_merge($lang, array(
+	'EXTIMGASLINK_CONFIG'			=> 'Allowed origin of images used in posts',
+	'EXTIMGASLINK_CONFIG_EXPLAIN'	=> 'Images, that are available through a secure connection, can be additionally allowed. Any other images are replaced with links. Images, that are available on the same domain as this board, are always allowed.',
+	'EXTIMGASLINK_INVALID_CONFIG'	=> 'The origin you specified is invalid.',
```

Comment:
Ich würde auch hier "permit" statt "allow" nutzen. Falls oben die Variante ohne "origin" gewählt wird würde ich zusätzlich aus "origin" in der Fehlermeldung "image origin" machen.